Sources/VitaminCore/Foundations/Assets/VitaminAssetsFlags.swift

```diff
@@ -13,3 +13,12 @@ extension VitaminAssets {
         VitaminAsset(name: countryCode.uppercased())
     }
 }
```

Comment:
I don't know where to write it, so I write it here : shouldn't the file be named `VitaminAssets+Flags.swift` to respect the convention we applied for extensions on system types ?
exemple : `UIImage+SwiftUI.swift` or `UIImage+Color.swift`

Really not blocking, you can leave it as is if you prefer, I just wanted to ask for potential future extension files.

Reply:
Yes, you're right, sorry for that too. I'll change it.